### PR TITLE
Update Hotjar destination docs

### PR DESCRIPTION
**Pull request by Maguna**
Reflect latest changes (events support), remove mentions of tools no longer available in Hotjar, among other updates.

### CODE REVIEW

File: src/connections/destinations/catalog/hotjar/index.md [hunk 1]
```diff
@@ -3,41 +3,36 @@ title: Hotjar Destination
...
-* [Target Polls and Incoming Feedback to users based on their User Attributes](https://help.hotjar.com/hc/en-us/articles/360022688554)
-* [Lookup and Delete user data based on their User ID](https://help.hotjar.com/hc/en-us/articles/360001749014)
-
-In time, most or all Hotjar features will use User Attributes in some way, with filtering Recordings planned in the first half of 2020.
-
-This destination is maintained by Segment.
+The Segment Hotjar Destination allows you to easily get started with Hotjar and its core APIs:
+1. Automatically install the [Hotjar Tracking Code](https://help.hotjar.com/hc/en-us/articles/115011639927);
+2. Automatically send [user attributes](https://help.hotjar.com/hc/en-us/articles/360033640653-Identify-API-Reference) to Hotjar by connecting your Segment `identify` calls with Hotjar's own Identify API;
+3. Automatically send [custom events](https://help.hotjar.com/hc/en-us/articles/4405109971095-Events-API-Reference) to Hotjar by connecting your Segment `track` calls with Hotjar's own Events API.
```
Comment (stayseesong) [inline src/connections/destinations/catalog/hotjar/index.md:11]:
```suggestion
1. Automatically install the [Hotjar Tracking Code](https://help.hotjar.com/hc/en-us/articles/115011639927){:target="_blank"}.
2. Automatically send [user attributes](https://help.hotjar.com/hc/en-us/articles/360033640653-Identify-API-Reference){:target="_blank"} to Hotjar by connecting your Segment `identify` calls with Hotjar's own Identify API.
3. Automatically send [custom events](https://help.hotjar.com/hc/en-us/articles/4405109971095-Events-API-Reference){:target="_blank"} to Hotjar by connecting your Segment `track` calls with Hotjar's own Events API.
```

File: src/connections/destinations/catalog/hotjar/index.md [hunk 2]
```diff
@@ -3,41 +3,36 @@ title: Hotjar Destination
...
 
+Knowing who your users are and what they're doing unlocks much more advanced filtering and targeting capabilities across all of Hotjar's tools, helping you find meaningful insights much faster.
```
Comment (stayseesong) [inline src/connections/destinations/catalog/hotjar/index.md:13]:
```suggestion
Knowing who your users are and what they're doing unlocks more advanced filtering and targeting capabilities across all of Hotjar's tools, helping you find meaningful insights faster.
```

File: src/connections/destinations/catalog/hotjar/index.md [hunk 3]
```diff
@@ -3,41 +3,36 @@ title: Hotjar Destination
...
 
 ## Getting Started
 
 {% include content/connection-modes.md %}
 
-1. From the Segment web app, click **Catalog**.
+1. From the Segment web app, click **Catalog**;
```
Comment (stayseesong) [inline src/connections/destinations/catalog/hotjar/index.md:19]:
```suggestion
1. Navigate to **Connections** and click **Add Destination** From the Segment web app.
```

File: src/connections/destinations/catalog/hotjar/index.md [hunk 4]
```diff
@@ -3,41 +3,36 @@ title: Hotjar Destination
...
 
-2. Search for "Hotjar" in the Catalog, select it, and choose which of your Javascript sources to connect the destination to.
+2. Search for "Hotjar" in the Catalog, select it, and choose which of your Javascript sources to connect the destination to;
```
Comment (stayseesong) [inline src/connections/destinations/catalog/hotjar/index.md:21]:
```suggestion
2. Search for *Hotjar* in the Catalog, select it, and choose the Javascript source you want to connect the destination to.
```

File: src/connections/destinations/catalog/hotjar/index.md [hunk 5]
```diff
@@ -3,41 +3,36 @@ title: Hotjar Destination
...
 
-3. Add your Hotjar Site ID to your Destination settings. You can find this under Settings and Sites & Organizations in your Hotjar dashboard. It should be a whole number (e.g. 123456).
+3. Add your **Hotjar Site ID** to your Destination settings. You can find this ID (e.g. 123456) in Account settings > Sites & Organizations;
```
Comment (stayseesong) [inline src/connections/destinations/catalog/hotjar/index.md:23]:
```suggestion
3. Add your **Hotjar Site ID** to your Destination settings. You can find this ID in Account settings > Sites & Organizations.
```

File: src/connections/destinations/catalog/hotjar/index.md [hunk 6]
```diff
@@ -3,41 +3,36 @@ title: Hotjar Destination
...
-
-Hotjar automatically starts tracking visitors based on the tools you have enabled in your Hotjar dashboard.
+4. Your changes will appear in the Segment CDN in about 45 minutes, and then Analytics.js will start asynchronously loading Hotjar's tracking snippet and sending data. If you are already using Hotjar, remove Hotjar's snippet from your code.
 
 ## Identify
 
-The Hotjar destination will automatically ingest a User ID, as well as values sent over your Identify spec as [traits](/docs/connections/spec/identify/#traits), as long as [User Attributes are enabled in Hotjar](https://help.hotjar.com/hc/en-us/articles/360038394053-How-to-Setup-User-Attributes-in-4-Steps#step-2-review-your-privacy-requirements-and-enable-user-attributes).
+The Hotjar destination will automatically ingest a User ID and any values sent over your Identify spec as [traits](/docs/connections/spec/identify/#traits), as long as session capture and user attributes are both enabled in Hotjar.
```
Comment (stayseesong) [inline src/connections/destinations/catalog/hotjar/index.md:29]:
```suggestion
The Hotjar destination automatically ingests a User ID and any values sent over your Identify spec as [traits](/docs/connections/spec/identify/#traits), as long as session capture and user attributes are both enabled in Hotjar.
```

File: src/connections/destinations/catalog/hotjar/index.md [hunk 7]
```diff
@@ -3,41 +3,36 @@ title: Hotjar Destination
...
 
 Identify calls that do not have a User ID value will not be sent to Hotjar.
 
 ### Nested values or lists
 
-The Hotjar Identify API is unable to ingest values passed as nested objects or lists over your `identify` Spec:
+As of July 2022, the Hotjar Identify API does not yet support ingesting values passed as nested objects or lists over your identify Spec:
```
Comment (stayseesong) [inline src/connections/destinations/catalog/hotjar/index.md:35]: Do we need to call out the date? Can't we just have it say: 
"The Hotjar Identify API doesn't support ingesting values passed as nested objects or lists over your identify Spec."

File: src/connections/destinations/catalog/hotjar/index.md [hunk 8]
```diff
@@ -54,6 +49,23 @@ The Hotjar Identify API is unable to ingest values passed as nested objects or l
     }
 ```
 
-In this example, Hotjar would reject all the values in the `address` field.
+In the example above, Hotjar would reject all the values in the `address` field.
```
Comment (stayseesong) [inline src/connections/destinations/catalog/hotjar/index.md:52]:
```suggestion
In the example above, Hotjar rejects all the values in the `address` field.
```

File: src/connections/destinations/catalog/hotjar/index.md [hunk 9]
```diff
@@ -54,6 +49,23 @@ The Hotjar Identify API is unable to ingest values passed as nested objects or l
...
+
+## Track
+
+The Hotjar destination will automatically ingest any user actions tracked over your Track spec as [events](/docs/connections/spec/track/), as long as session capture is enabled in Hotjar.
```
Comment (stayseesong) [inline src/connections/destinations/catalog/hotjar/index.md:56]:
```suggestion
The Hotjar destination automatically ingests any user actions tracked over your Track spec as [events](/docs/connections/spec/track/), as long as session capture is enabled in Hotjar.
```

**Review by stayseesong — CHANGES_REQUESTED** [review]

File: src/connections/destinations/catalog/hotjar/index.md [hunk 10]
```diff
@@ -54,6 +49,23 @@ The Hotjar Identify API is unable to ingest values passed as nested objects or l
...
+
+### Event properties
+
+As of July 2022, the Hotjar Events API does not yet support ingesting event properties:
```
[the remark this answers is not in the document]
Comment (Maguna) [reply src/connections/destinations/catalog/hotjar/index.md:60]: What about something like "Currently, the Hotjar Events API doesn't support ingesting event properties:"

But don't feel strongly about it

**Review by Maguna — COMMENTED** [review]
All looks good, thanks for the quick review 👍

File: src/connections/destinations/catalog/hotjar/index.md (hunk 7 above)
Comment (Maguna) [reply src/connections/destinations/catalog/hotjar/index.md]: Updated the copy

**Review by markzegarelli — APPROVED** [review]

**Comment by markzegarelli** [comment]
Thanks @Maguna! We'll roll these changes out with our next deploy on July 5.

**Review by markzegarelli — APPROVED** [review]

**Comment by Maguna** [comment]
@markzegarelli hey quick question could we remove this as well from the page - and ideally from the in-app wizard as well? We don't support it anymore.
image

**Pull request merged**